site/styles/_shift72.scss

```diff
@@ -11,8 +11,12 @@
   }
 }
 
+// exists once in <s72-searchresults/>
 .s72-btn-search {
   @extend .btn-primary-override;
+  font-size: 24px;
+  height: var(--input-height);
```

Comment:
`height` doesn't seem to do anything here.

Reply:
- removed